### PR TITLE
SDL_sound

### LINKED ISSUE #12: mounting CUE/BIN image is not supported
Probably due DOSBox was compiled without SDL_sound library, which is essential for this functionality.

Sample .CUE file:
```
$ cat lba2.dat 
FILE "LBA2.GOG" BINARY
  TRACK 01 MODE1/2352
    INDEX 01 00:00:00
FILE "LBA2.OGG" MP3
  TRACK 02 AUDIO
      INDEX 01 00:00:00

### PR DESCRIPTION
**Pull request by bam80**
Fixes #12

Maybe better bundle this instead of creating a new module: 
https://github.com/flathub/shared-modules/issues/41

Couldn't use official sources here:
https://github.com/flatpak/flatpak-builder/issues/250

### CODE REVIEW

File: com.dosbox.DOSBox.json [hunk 1]
```diff
@@ -26,6 +26,17 @@
         "shared-modules/glu/glu-9.0.0.json",
         "shared-modules/SDL/SDL-1.2.15.json",
         "shared-modules/SDL/SDL_net-1.2.8.json",
+        {
+            "name": "SDL_sound",
+            "config-opts": ["--disable-static"],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/sezero/SDL_sound",
```
Comment (AsciiWolf) [inline com.dosbox.DOSBox.json]: Hmm, I am not sure if I like using some unofficial fork repo. I think we should wait for the [download issue](https://github.com/flatpak/flatpak-builder/issues/250) to be fixed instead.

Comment (bam80) [reply com.dosbox.DOSBox.json]: I don't like it also, but actually that fork contains fully unmodified branches/tags from official repo, so we could rely on that for a while

Comment (bam80) [reply com.dosbox.DOSBox.json]: We could compare checkouts of that tag from git vs mercurial, to be sure they are identical.
@AsciiWolf would it be enough to be accepted?

Comment (AsciiWolf) [reply com.dosbox.DOSBox.json]: Why not using the official archive instead? You can use some mirror as noted in my comment [here](https://github.com/flatpak/flatpak-builder/issues/250#issuecomment-452115038) if the official one doesn't work for some reason.

Comment (bam80) [reply com.dosbox.DOSBox.json]: Still doesn't work for me, but buildbot seems hasn't the problem with official sources

**Comment by AsciiWolf** [comment]
testbot, build

**Comment by AsciiWolf** [comment]
bot, build

**Comment by AsciiWolf** [comment]
Build [fails](https://flathub.org/builds/#/builders/39/builds/669) on aarch64. Try adding `"rm-configure": true`.

**Comment by AsciiWolf** [comment]
Hmm, it seems that it didn't help. :-( Try  `"no-autogen": true` instead.

**Comment by AsciiWolf** [comment]
bot, build

**Comment by AsciiWolf** [comment]
LGTM!

**Pull request merged**